mlflow/store/tracking/gateway/abstract_mixin.py

```diff
@@ -0,0 +1,391 @@
+from typing import Any
+
+from mlflow.entities import (
+    GatewayEndpoint,
+    GatewayEndpointBinding,
+    GatewayEndpointModelMapping,
+    GatewayModelDefinition,
+    GatewaySecret,
+)
+from mlflow.store.tracking.gateway.entities import GatewayEndpointConfig
+
+
+class GatewayStoreMixin:
```

Comment:
Did we split these methods as a Mixin because tracking store is getting heavy?

Reply:
Mostly to separate out the ambiguous terms "model" (in MLflow that means something different in our public APIs), but also, yeah, sqlalchemy_store.py is the Titanic of Python modules

Reply:
Make sense, we should probably do some refactoring of tracking stores using mixin